src/contexts/interface/Interface.tsx

```diff
@@ -11,13 +11,15 @@ export interface InterfaceContextContextProps {
   showMobileToolbar: boolean;
   showWalletList: boolean;
   transactionsTabIsOpen: boolean;
+  transactionComplete: boolean;
```

Comment:
Interface is only for things related to interface, it's not for individual transactions.